assets/blueprints/flash_loan/src/lib.rs

```diff
@@ -10,7 +10,7 @@ mod basic_flash_loan {
     struct BasicFlashLoan {
         loan_vault: Vault,
         auth_vault: Vault,
-        transient_resource_address: ResourceAddress,
+        transient_resource_manager: ResourceManager,
```

Comment:
From scrypto point of view I was reserving `Global`/`Owned` to be only used for components since Scrypto sees resources and packages as special things.

For example, if we went with `Global<ResourceManager>` I'd also expect `Owned<Vault>` and `Owned<Bucket>` but I don't think we want to go down that path.